apple-silicon-support/modules/mesa/default.nix

```
@@ -37,7 +37,8 @@
       # (and in a way compatible with pure evaluation)
       nixpkgs.overlays = [
         (final: prev: {
-          mesa = final.mesa-asahi-edge;
+          #  revent cross-built Mesas that might run on this system under emulation (e.g. through Steam and box64) from using the special Asahi Mesa
```

Copy-paste error and rephrasing, thanks again.

```suggestion
          # prevent cross-built Mesas that might be evaluated using this config (e.g. Steam emulation via box64) from using the special Asahi Mesa
```

---

Done :upside_down_face: